### ISSUE TITLE
Recurrent Reinforcement Learning Issue

### ISSUE THREAD

**Issue by MAngO311** [tensorflow/tensorflow#12072]
Hi guys,

I am working on a structure with recurrent neural network as the deep part of deep reinforcement learning.

I have been searching all along but I haven't found a way to build this structure by using tensorflow. The recurrent feature makes the objective function iterative. I am wondering if TF can calculate the gradient by itself.

It would be great if I can get to know if this structure can be done or not.

**Comment by rohan100jain** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
